astlib/ast_mapper.mli

```diff
@@ -1,3 +1,5 @@
+(** Meta data related interface for a ppx rewriter *)
```

Comment:
Maybe this module should be renamed as it doesn't have much in common with the original Ast_mapper anymore!

Reply:
Yes, that's true, even this documentation shows that it's not really related to `Ast_mapper` anymore. I've renamed it to `Ast_metadata`. Thanks!